packages/zwave-js/src/lib/commandclass/MultilevelSwitchCC.ts

```diff
@@ -98,6 +99,14 @@ function getCurrentValueValueID(endpoint: number): ValueID {
 	};
 }
 
+/** Returns the ValueID used to remember whether a node supports supervision on the start/stop level change commands*/
+export function getSuperviseStartStopLevelChangeValueId(): ValueID {
```

Comment:
@AlCalzone Is this internal right?

Reply:
Ah yeah

Reply:
@AlCalzone I mean the valueId will not be crated right? It's an internal valueid? Does it need @internal ?

Reply:
This is one of the weirder parts of value IDs. The function won't be exported now. The valueID is marked internal in the CC constructor (`registerValue`, boolean parameter). One of the many aspects to be considered in #552

Reply:
Not 100% clear to me but ok I trust you :laughing: